### PR TITLE
Add more info about -additional-config

### PR DESCRIPTION
**Pull request by renetapopova**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1945.surge.sh

Updated pages:
- [Aggregate a database backup chain](https://neo4j-docs-operations-1945.surge.sh/operations-manual/5/backup-restore/aggregate)
- [Copy a database store](https://neo4j-docs-operations-1945.surge.sh/operations-manual/5/backup-restore/copy-database)
- [Inspect the metadata of a backup file](https://neo4j-docs-operations-1945.surge.sh/operations-manual/5/backup-restore/inspect)
- [Back up an offline database](https://neo4j-docs-operations-1945.surge.sh/operations-manual/5/backup-restore/offline-backup)
- [Back up an online database](https://neo4j-docs-operations-1945.surge.sh/operations-manual/5/backup-restore/online-backup)
- [Restore a database backup](https://neo4j-docs-operations-1945.surge.sh/operations-manual/5/backup-restore/restore-backup)
- [Restore a database dump](https://neo4j-docs-operations-1945.surge.sh/operations-manual/5/backup-restore/restore-dump)
- [Consistency checker](https://neo4j-docs-operations-1945.surge.sh/operations-manual/5/tools/neo4j-admin/consistency-checker)
- [Neo4j Admin and Neo4j CLI](https://neo4j-docs-operations-1945.surge.sh/operations-manual/5/tools/neo4j-admin)

File: modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc [hunk 1]
```diff
@@ -131,7 +131,7 @@ For more information, please contact Neo4j Professional Services.
 | Description
 | Default
 
-|--additional-config=<file>
+--additional-config=<file>footnote:[See xref:tools/neo4j-admin/index.adoc#_configuration[Tools -> Configuration] for details.]
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc:134]:
```suggestion
|--additional-config=<file>footnote:[See xref:tools/neo4j-admin/index.adoc#_configuration[Tools -> Configuration] for details.]
```

Comment (renetapopova) [reply modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc:134]: Thanks! I was looking for it. 😄

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1945.surge.sh

Updated pages:
- [Aggregate a database backup chain](https://neo4j-docs-operations-1945.surge.sh/operations-manual/5/backup-restore/aggregate)
- [Copy a database store](https://neo4j-docs-operations-1945.surge.sh/operations-manual/5/backup-restore/copy-database)
- [Inspect the metadata of a backup file](https://neo4j-docs-operations-1945.surge.sh/operations-manual/5/backup-restore/inspect)
- [Back up an offline database](https://neo4j-docs-operations-1945.surge.sh/operations-manual/5/backup-restore/offline-backup)
- [Back up an online database](https://neo4j-docs-operations-1945.surge.sh/operations-manual/5/backup-restore/online-backup)
- [Restore a database backup](https://neo4j-docs-operations-1945.surge.sh/operations-manual/5/backup-restore/restore-backup)
- [Restore a database dump](https://neo4j-docs-operations-1945.surge.sh/operations-manual/5/backup-restore/restore-dump)
- [Consistency checker](https://neo4j-docs-operations-1945.surge.sh/operations-manual/5/tools/neo4j-admin/consistency-checker)
- [Neo4j Admin and Neo4j CLI](https://neo4j-docs-operations-1945.surge.sh/operations-manual/5/tools/neo4j-admin)

**Review by NataliaIvakina — APPROVED** [review]
Looks good!

**Review by PetrJanouch — APPROVED** [review]
nice

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1945.surge.sh

Updated pages:
- [Aggregate a database backup chain](https://neo4j-docs-operations-1945.surge.sh/operations-manual/5/backup-restore/aggregate)
- [Copy a database store](https://neo4j-docs-operations-1945.surge.sh/operations-manual/5/backup-restore/copy-database)
- [Inspect the metadata of a backup file](https://neo4j-docs-operations-1945.surge.sh/operations-manual/5/backup-restore/inspect)
- [Back up an offline database](https://neo4j-docs-operations-1945.surge.sh/operations-manual/5/backup-restore/offline-backup)
- [Back up an online database](https://neo4j-docs-operations-1945.surge.sh/operations-manual/5/backup-restore/online-backup)
- [Restore a database backup](https://neo4j-docs-operations-1945.surge.sh/operations-manual/5/backup-restore/restore-backup)
- [Restore a database dump](https://neo4j-docs-operations-1945.surge.sh/operations-manual/5/backup-restore/restore-dump)
- [Consistency checker](https://neo4j-docs-operations-1945.surge.sh/operations-manual/5/tools/neo4j-admin/consistency-checker)
- [Neo4j Admin and Neo4j CLI](https://neo4j-docs-operations-1945.surge.sh/operations-manual/5/tools/neo4j-admin)

**Comment by renetapopova** [comment]
The failures will be fixed by #1858.

**Pull request merged**